atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/sweep/asts/ShardedSweepableBucketRetrieverTest.java

```diff
@@ -52,10 +53,11 @@ public class ShardedSweepableBucketRetrieverTest {
             SweepTimestamps.builder().sweepTimestamp(1).lastSweptTimestamp(12).build();
     private final SettableRefreshable<Integer> numShards = Refreshable.create(1);
     private final SettableRefreshable<Integer> maxParallelism = Refreshable.create(1);
-    private final SettableRefreshable<Duration> maxBackoff = Refreshable.create(Duration.ofMillis(0));
     private final TestShardedRetrievalStrategy strategy = new TestShardedRetrievalStrategy();
     private final TestParallelTaskExecutor parallelTaskExecutor = new TestParallelTaskExecutor();
     private final ExecutorService executorService = PTExecutors.newSingleThreadScheduledExecutor();
+    private final AtomicBoolean wasSleepCalled = new AtomicBoolean(false);
```

Comment:
Copying concern:
wasSleepCalled in tests - it's only read once in the tests, didn't want to mock unnecessarily, but figured I might as well inject it for all rather than have two separate functions. If you feel strongly, I can just switch the default for tests to be () -> {} and recreate the retriever as needed in the new test.